### ISSUE TITLE
[Task]: Rewrite most of the client declaration tests to point to a local file

### ISSUE THREAD

**Issue by MaryamZi** [ballerina-platform/ballerina-lang#38205]
### Description

$title. Currently almost all are remote, which cause intermittent failures at times.

By default, it would be better to not be remote.

### Related area

-> Compilation

**Comment by hasithaa** [comment]
@MaryamZi We can close this issue, since this is no longer valid. isn't it?

**Comment by MaryamZi** [comment]
> @MaryamZi We can close this issue, since this is no longer valid. isn't it?

Yes, closing since no longer valid (because the client declaration was removed).
